python/lsst/sims/maf/metrics/hourglassMetric.py

```diff
@@ -35,7 +35,7 @@ def run(self, dataSlice, slicePoint=None):
                  'twi12_set', 'twi18_rise', 'twi18_set']
         types = ['float']*len(names)
         pernight = np.zeros(len(unights), dtype=list(zip(names, types)))
-        pernight['mjd'] = dataSlice['observationStartMJD'][uindx]
+        pernight['mjd'] = dataSlice[self.mjdcol][uindx]
```

Comment:
I don't understand what's going on here. In LSST master, this is already fixed. (and we'd like to use mjdCol to match LSST's (old) standard for camelcase). 
Same wither filterCol and nightCol.